ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -7171,3 +7171,13 @@ FPHC:
   uses:
   - heating_water_valve_percentage_command
   - chilled_water_valve_percentage_command
+
+CWSWC:
```

Comment:
I would instead make `condensing_water_...` = `steam_[supply/return]_...`

I understand why you would use condensing (since the steam probably does condense as it moves through the HX), but for this model we should use `steam` instead. Please specify if the steam isolation valve is on the supply or return side, as well. 

Do you have other points available to you? Like the steam temperature or pressure?